test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/EmailAllowedListSpec.scala

```diff
@@ -20,18 +20,18 @@ import org.scalatest.matchers.must.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 import org.scalatestplus.mockito.MockitoSugar
 
-class UtrAllowedListSpec extends AnyWordSpec with Matchers with MockitoSugar {
+class EmailAllowedListSpec extends AnyWordSpec with Matchers with MockitoSugar {
 
   "utr allow list" when {
```

Comment:
Needs changing to read "email allow list".

Reply:
This has been addressed.